lib/rules/best-practises/no-console.js

```diff
@@ -17,11 +17,11 @@ const meta = {
         },
         {
           description: 'No hardhat/console.sol import statements',
-          code: 'import "hardhat/console.sol"' // eslint-disable-line
+          code: 'import "hardhat/console.sol"', // eslint-disable-line
         },
         {
           description: 'No forge-std console.sol & console2.sol import statements',
-          code: 'import "forge-std/consoleN.sol"' // eslint-disable-line
+          code: 'import "forge-std/consoleN.sol"', // eslint-disable-line
```

Comment:
I didn't notice this in the previous review, but this `// eslint-disable-line` comments shouldn't be used unless there is a good reason. It seems that in the previous implementation they were used to fix the warning about the missing comma, and now they were kept after the proper fix was done.

Reply:
done